diagnostic.md

```diff
@@ -32,7 +42,9 @@ y = 10;
 
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
-<!-- solution below -->
+after 'x=y', x = 3
+after 'y=10', y = 10
+The variables themselves don't change but their values do
 
```

Comment:
I'll give you this one, but please note that you didn't give the values of x and y for the 1st two lines!